CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ celeritas_optional_package(OpenMP "Enable CPU shared-memory parallelism")
 celeritas_optional_package(Python "Use Python to generate and preprocess")
 celeritas_optional_package(ROOT "Enable ROOT I/O")
 celeritas_optional_package(VecGeom "Use VecGeom geometry")
+celeritas_optional_package(Perfetto "Perfetto tracing library")
```

Comment:
Since it sounds like perfetto isn't ever externally installable, I think this should just be an `option(CELERIITAS_USE_Perfetto` like we (will) have for JSON.